build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -ue
+
+# Expect the following envvars to be set:
+# - APP
+# - VERSION
+# - COMMIT
+# - TARGET_OS
+# - LEDGER_ENABLED
+# - DEBUG
+
+# Source builder's functions library
+. /usr/local/share/cosmos-sdk/buildlib.sh
```

Comment:
Is this `build.sh` script actually needed, and why is it referencing Cosmos SDK stuff?

Reply:
This is because the docker image stores the copied data in cosmos-sdk. This needs to be modified. I believe this script can be moved into the docker container

Reply:
Yeah I can change the installaton path into a non-cosmos sdk related alternative one